dockernet/config/relayer_config_dydx_noble.yaml

```diff
@@ -34,6 +34,21 @@ chains:
       timeout: 20s
       output-format: json
       sign-mode: direct
+  osmo:
```

Comment:
Where can I see which version of Osmosis we have pulled into dockernet? Relevant because the GAMM module which we do the trade ICA tx at has slightly different APIs at different versions

Reply:
[Here's](https://github.com/Stride-Labs/stride/blob/main/dockernet/dockerfiles/Dockerfile.osmo#L7) the version we use at the moment - but these dydx branches are a little stale and I think use a different version atm. I'll refresh everything after merging phase 1/3 to main